README.md

```diff
@@ -55,8 +61,8 @@ Run `git clone --recursive https://github.com/discordpp/echo-bot.git` in your de
  1. Create a `build` directory (folder) with `mkdir build`
  2. Enter the folder with `cd build`
  3. Go back to your Application's 'bot' page from before, and copy the token. Run `echo Bot [token] > token.dat`, replacing the brackets and the word token
- 4. Build the makefiles with `cmake ../`
- 5. Build the bot with `make`
+ 4. Build the makefiles with `cmake ../` (If you are using vcpkg then `cmake -DCMAKE_TOOLCHAIN_FILE="your vcpkg.cmake folder" ../`)
```

Comment:
Maybe add the `vcpkg` stuff as an indented subpoint of this one instead of in parenthesis?